Gems/ImGui/Code/Include/ImGuiBus.h

```diff
@@ -94,7 +92,7 @@ namespace ImGui
         virtual void ToggleToImGuiVisibleState(DisplayState state) = 0;
         virtual void SetDpiScalingFactor(float dpiScalingFactor) = 0;
         virtual float GetDpiScalingFactor() const = 0;
-        virtual void Render() = 0;
+        virtual ImDrawData* GetImguiDrawData() = 0;
```

Comment:
We should probably bump the ImGui `gem.json` Major version because this looks like an API breaking change.

Reply:
I can easily bump the major version, currently 0.0.0 to 1.0.0. What knock-ons might I expect from that change however? I'm not too familiar with what we have in terms of GEM dependency management, nor whether a version bump like this may require dependency changes elsewhere.

Reply:
It's safe to do, and mainly important for external devs. details here https://www.o3de.org/docs/user-guide/gems/gem-versioning/